k8s/autopilot/autopilotruleobject.go

```diff
@@ -0,0 +1,59 @@
+package autopilot
+
+import (
+	autv1alpha1 "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RuleObjectOps is an interface to perform k8s AutopilotRuleObjects operations
+type RuleObjectOps interface {
+	// CreateAutopilotRuleObject creates the AutopilotRuleObject object
+	CreateAutopilotRuleObject(object *autv1alpha1.AutopilotRuleObject) (*autv1alpha1.AutopilotRuleObject, error)
+	// GetAutopilotRuleObject gets the AutopilotRuleObject for the provided name
+	GetAutopilotRuleObject(string) (*autv1alpha1.AutopilotRuleObject, error)
+	// UpdateAutopilotRuleObject updates the AutopilotRuleObject
+	UpdateAutopilotRuleObject(*autv1alpha1.AutopilotRuleObject) (*autv1alpha1.AutopilotRuleObject, error)
+	// DeleteAutopilotRuleObject deletes the AutopilotRuleObject of the given name
+	DeleteAutopilotRuleObject(string) error
```

Comment:
same here, add parameter name. That improves readability

Reply:
done